src/client/protocol.h

```diff
@@ -90,8 +93,16 @@ struct client_file
  * in milliseconds. */
 void clientContextMillis(struct client_context *context, long millis);
 
+void *mallocChecked(size_t);
+void *callocChecked(size_t, size_t);
+char *strdupChecked(const char *);
+char *strndupChecked(const char *, size_t);
+
+void makeValueOwned(struct value *);
+void freeOwnedValue(struct value);
```

Comment:
At the moment these functions don't seem to be used outside of `protocol.c`. Is there a plan to use them elsewhere too?